users/vieting/experiments/switchboard/ctc/feat/experiments.py

```diff
@@ -719,41 +724,354 @@ def run_scf_baseline():
         dev_corpora,
     ) = get_datasets()
     returnn_args = {
-        "batch_size": 5000,
+        "batch_size": 7000,
```

Comment:
I changed the baseline from 2x5k to 7k to be in line with the audio perturbation experiments with 7k. 5k does not work, because the trainings do not converge.

Reply:
But why not 2x5k? I'm not comfortable with changing the baseline unless there is evidence to do it.